pkg/installer/install.go

```diff
@@ -49,6 +49,43 @@ func (m *manager) Manifests(ctx context.Context) (graph.Graph, error) {
 	return g, err
 }
 
+func (m *manager) Ignition(ctx context.Context) (graph.Graph, error) {
+	var (
+		installConfig *installconfig.InstallConfig
+		image         *releaseimage.Image
+		g             graph.Graph
+	)
+
+	s := []steps.Step{
+		steps.Action(func(ctx context.Context) error {
+			var err error
+			installConfig, image, err = m.generateInstallConfig(ctx)
+			return err
+		}),
+
+		steps.Action(func(ctx context.Context) error {
+			var err error
+			// Applies ARO-specific customisations to the InstallConfig
+			g, err = m.applyInstallConfigCustomisations(installConfig, image)
+			return err
+		}),
+
+		steps.Action(func(ctx context.Context) error {
+			var err error
+			// Applies ARO-specific customisations to the InstallConfig
+			g, err = m.applyIgnitionConfigCustomisations(g)
```

Comment:
Testing a mixture of existing and new code is confusing. Our immediately goal here is to characterise the existing code.